<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EBaseWidget</class>
 <widget class="QWidget" name="EBaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EBaseWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lLastName">
     <property name="text">
      <string>Prenom:</string>
     </property>
     <property name="buddy">
      <cstring>leLastName</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="leLastName"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="LDateNaiss">
     <property name="text">
      <string>Date de Naissance:</string>
     </property>
     <property name="buddy">
      <cstring>sbDay</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lAdresse">
     <property name="text">
      <string>Adresse:</string>
     </property>
     <property name="buddy">
      <cstring>leAdresse</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="leAdresse"/>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLabel" name="lEtat">
     <property name="text">
      <string>Etat (information supplementaire):</string>
     </property>
     <property name="buddy">
      <cstring>pteEtat</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPlainTextEdit" name="pteEtat"/>
   </item>
   <item row="3" column="3">
    <layout class="QHBoxLayout" name="DateNaissLayout">
     <item>
      <widget class="QSpinBox" name="sbDay"/>
     </item>
     <item>
      <widget class="QComboBox" name="sbMonth"/>
     </item>
     <item>
      <widget class="QSpinBox" name="sbYear"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>Nom:</string>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leLastName</tabstop>
  <tabstop>sbDay</tabstop>
  <tabstop>sbMonth</tabstop>
  <tabstop>sbYear</tabstop>
  <tabstop>leAdresse</tabstop>
  <tabstop>pteEtat</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
